include/RAJA/pattern/kernel/internal.hpp

```diff
@@ -258,10 +258,9 @@ RAJA_HOST_DEVICE RAJA_INLINE void invoke_lambda_expanded(
     camp::idx_seq<ParamIdx...> const &,
     Data &&data)
 {
-  camp::get<LoopIndex>(
-      data.bodies)((camp::get<OffsetIdx>(data.segment_tuple)
-                        .begin()[camp::get<OffsetIdx>(data.offset_tuple)])...,
-                   camp::get<ParamIdx>(data.param_tuple)...);
+  camp::get<LoopIndex>(data.bodies)
+    ((camp::get<OffsetIdx>(data.segment_tuple).begin()[camp::get<OffsetIdx>(data.offset_tuple)])...,
```

Comment:
@ajkunen , by OffsetIdx this corresponds to the iteration space index right?  For example it can be a RangeSegment or ListSegment.

Reply:
I used SegIdx in other places, I can make it consistent though.

Reply:
That's right, it's an offset into the segment tuple... So it can be anything segment type

Reply:
SegIdx would be more descriptive, feel free to change OffsetIdx to SegIdx to make things more consistent.